go.mod

```diff
@@ -3,3 +3,10 @@ module github.com/alecthomas/protobuf
 go 1.17
 
 require github.com/alecthomas/participle/v2 v2.0.0-alpha7
+
+require (
+	github.com/google/go-cmp v0.5.5
+	google.golang.org/protobuf v1.27.1
+)
```

Comment:
:( Only used in tests, I tried without but couldn't make much sense of the FileDescriptor diffs.